solution/zenko-version-cr.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: zenko.io/v1alpha1
+kind: ZenkoVersion
```

Comment:
What is this for? Where does it come from?
Is it supposed to describe a Zenko release? If yes, maybe the Solution manifest is sufficient?

Reply:
This file was removed